src/rhsmlib/dbus/base_object.py

```diff
@@ -13,62 +13,63 @@
 #
 import logging
 import dbus.service
-
-from rhsmlib.dbus import constants, exceptions, util
+from typing import TYPE_CHECKING
 
 from subscription_manager import utils
 from subscription_manager import injection as inj
 from subscription_manager.injectioninit import init_dep_injection
 
+from rhsmlib.dbus import constants, exceptions, util
 from rhsmlib.services import config
 import rhsm.config
 
+if TYPE_CHECKING:
+    from dbus.service import BusName
+    from rhsm.connection import UEPConnection
+    from subscription_manager.cp_provider import CPProvider
+    from subscription_manager.identity import Identity
+
 init_dep_injection()
 
 log = logging.getLogger(__name__)
 
 
-class BaseObject(dbus.service.Object):
```

Comment:
Just not: The diff of GitHub is sometimes so confusing...  This class was not removed.